src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -219,7 +298,42 @@ protected void LookupMembersInType(LookupResult result, TypeSymbol type, string
 
             // TODO: Diagnose ambiguity problems here, and conflicts between non-method and method? Or is that
             // done in the caller?
+
+            if ((options & LookupOptions.SearchInExtensionTypes) != 0 && !result.IsMultiViable)
```

Comment:
>IsMultiViable [](http://example.com/codeflow?start=81&length=13)

Are ambiguity cases considered ```IsMultiViable```?

Reply:
Yes. For instance, in test `ExtensionMemberLookup_Simple_Static_MethodGroupExists`, we'll get here with `IsMultiViable == true` and two symbols in the result.

Reply:
Could you clarify what scenario you have in mind? Sorry, I didn't follow

Reply:
> Could you clarify what scenario you have in mind? Sorry, I didn't follow

For example, there are two fields with the dame name in the type (valid in metadata). What happens for the scenario trying to access a field?

Reply:
If I try to use `CreateCompilationWithIL` to create metadata for a type with duplicate fields (such as the one below), I get an error: `{"The provided IL cannot be compiled.  C:\\Windows\\Microsoft.NET\\Framework64\\v4.0.30319\\ilasm.exe ...:  error : Duplicate field declaration: 'member'`.
Is there a secret handshake to create such a type?

```
.class public auto ansi beforefieldinit C
    extends [System.Runtime]System.Object
{
    .field public int32 'member'
    .field public int32 'member'
}
```

Reply:
> Is there a secret handshake to create such a type?

No, just use different signatures. For example, fields' types could be different, or custom modifiers are different, or a combination of both is different.

Reply:
Added a test for this (`ExtensionMemberLookup_DuplicateFieldNamesInUnderlyingType`) and confirmed that `IsMultiViable == true` in that scenario.